File: ex7/ex.py
import smtplib

server = smtplib.SMTP('smtp.example.com', 587)
server.starttls()
server.login("user@example.com", "password")
server.sendmail("from@example.com", "to@example.com", 
    "Subject: Alerte\n\nMessage d'alerte")

File: ex7/ex.go
package main

import (
    "net/smtp"
)

func main() {
    msg := []byte("Subject: Alerte système\r\n\r\nAlerte: problème détecté.\r\n")
    smtp.SendMail("smtp.gmail.com:587",
        smtp.PlainAuth("", "admin@example.com", "motdepasse", "smtp.gmail.com"),
        "admin@example.com", []string{"user@example.com"}, msg)
}
